src/packages/@ncigdc/utils/constants.js

```diff
@@ -816,7 +816,7 @@ export const DEV_USER_CA = [ // controlled access mock
 
 export const AWG_TOKEN_EXPIRY = (
   localStorage.AWG_TOKEN_EXPIRY ||
```

Comment:
I'm not sure if you might want the local storage variable to have the same prefix for consistency, but it shouldn't affect the deployment at all, so I don't see an issue either way.

Reply:
Thought I'd keep it without prefix for ease of use while debugging, but you're right... it should be consistent.